### DISCUSSION TITLE
Help with hanging "Load GraphQL schemas" step in cli

### DISCUSSION THREAD

**Discussion by bazzyb** [dotansimha/graphql-code-generator#10022] [General]
For the last month or 2, we've had an issue with the "Load GraphQL schemas" step getting stuck in the CI pipeline. It never fails locally, and only fails 1 in 10 times in the pipeline. 

I don't specifically think it's an issue with the library, however what I can't get is any sort of useful way of debugging the issue. The pipeline eventually times out after an hour, and no useful logs are output. 

I added `debug: true` and `verbose: true` to the codegen config, but it did nothing for the logs. 

Anyone have any idea how we could go about debugging this issue?

Below is the output in the pipeline:
```
$ npm run codegen
> graphql-codegen --config codegen.ts
[STARTED] Parse Configuration
[SUCCESS] Parse Configuration
[STARTED] Generate outputs
[STARTED] Generate to src/graphql/schema.graphql
[STARTED] Generate to src/types.ts
[STARTED] Generate to src/
[STARTED] Load GraphQL schemas
[STARTED] Load GraphQL schemas
[STARTED] Load GraphQL schemas
WARNING: step_script could not run to completion because the timeout was exceeded. For more control over job and script timeouts see: https://docs.gitlab.com/ee/ci/runners/configure_runners.html#set-script-and-after_script-timeouts
ERROR: Job failed: execution took longer than 1h0m0s seconds
```

**Comment by stevenlemon-sh** [comment]
Did you ever figure out what caused it?
We've just started having the same issue.

**Reply by bazzyb** [reply]
Unfortunately we never narrowed down what was causing the issue, it just stopped happening after a month or 2!
